readthedocs_ext/readthedocs.py

```diff
@@ -250,7 +250,21 @@ def remove_search_init(app, exception):
     )
 
     if os.path.exists(searchtools_file):
-        replacement_text = '/* Search initialization removed for Read the Docs */'
+        replacement_text = r"""
+/* Search initialization manipulated by Read the Docs */
+/* See https://github.com/readthedocs/addons/issues/213 for more information */
+
+function triggerSearch() {
+  const addonsInjected = document.querySelector(
+          'script[src="/_/static/javascript/readthedocs-addons.js"]'
```

Comment:
It would be useful if there was a better way to detect this, but seems fine for now..

Reply:
Yeah, I didn't find a different/better way to do this. We could eventually subscribe to our `CustomEvent` when it's deployed. This PR is a workaround for now, since the real fix is not installing this extension anymore 😄 --but we are not there yet.